src/fix_deposit.cpp

```diff
@@ -227,37 +227,31 @@ void FixDeposit::init()
 {
   // set index and check validity of region
 
-  iregion = domain->find_region(idregion);
-  if (iregion == -1)
-    error->all(FLERR,"Region ID for fix deposit does not exist");
+  iregion = domain->get_region_by_id(idregion);
```

Comment:
rename to `region`?

Reply:
I have thought about this (there are lots more cases like that for fixes and computes, often in loops, though), but decided to keep the variable name to avoid any confusion with how the Pointers class provides access to other "base" classes (doesn't apply to fixes, computes, or regions, because there can be multiple, but still).